### ISSUE TITLE
Replay desyncs when using higher speed than 100

### ISSUE THREAD

**Issue by MBilderbeek** [openMSX/openMSX#1422]
A replay of a turboR or an MSX2 which runs e.g. The Maze of Galious Enhanced (see attachment) desyncs when running at a different speed than 100 (tested: fast forward (F9) and speed 200).  At fast forward speed the attached example desyncs almost immediately.

I also tried an MSX1 with the original MoG game, and that did not desync. A quick run of Vampire Killer on a 8250 also did not desync.

(Will do more testing later to find out what is the factor that makes it desync...)
[test.xml.gz](https://github.com/openMSX/openMSX/files/8809413/test.xml.gz)

**Comment by MBilderbeek** [comment]
Some new findings:
- I cannot reproduce it with the original MoG or any other game I tried (Space Manbow)
- Replaying old replay files I had (of other games) worked fine.
- I can also reproduce this issue with openMSX 17.0

One thing I found weird in the replay file is that it records an "SRAM" with the game, which is a KonamiSCC mapper game:
```
                                <item version="2">
                                  <name>mappertype</name>
                                  <data>KonamiSCC</data>
                                  <attributes/>
                                  <children/>
                                </item>
                                <item version="2">
                                  <name>sramname</name>
                                  <data>mog104.rom.SRAM</data>
                                  <attributes/>
                                  <children/>
                                </item>
```
Probably not related, but I don't understand what it is for.

**Comment by m9710797** [comment]
Probably fixed in 910a58ad1a9d144607bcea0711071f87b4a4acd3.
@MBilderbeek can you verify?

**Comment by MBilderbeek** [comment]
Verified, I can't reproduce this anymore. Thanks! :) Nice find anyway!
